libs/qlik/src/dashboard-page/vl-qlik-dashboard-page.component.ts

```diff
@@ -0,0 +1,267 @@
+import { CSSResult, html, PropertyDeclarations } from 'lit';
+import { BaseLitElement, registerWebComponents, webComponent } from '@domg-wc/common-utilities';
+import { vlElementsStyle } from '@domg-wc/elements';
+import '../dashboard';
```

Comment:
Gelieve een names import te doen van het Dashboard component en dit component te registreren in de `registerWebComponents`.